### ISSUE TITLE
AppImage doesn't launch and AUR install can't connect to downloads.sentry-cdn.com

### ISSUE THREAD

**Issue by owosoup** [spacedriveapp/spacedrive#1929]
### Describe the bug

I tried installing spacedrive, first from the AppImage file itself, the intended way. Upon executing it, nothing happens. 
So I tried installing the latest AUR version; and got this error message during installation:
![image](https://github.com/spacedriveapp/spacedrive/assets/95828007/aa01fd13-aedc-49a6-bb0a-5cbf437c271a)

### Reproduction

1.use Manjaro linux, KDE Plasma and pamac/intergated software manager (same result)
2. download alpha 0.1.4 AppImage / install from command line / software manager
3. it doesn't work.

### Expected behavior

I can install spacedrive.

### Platform and versions

```Shell
8.13.1
cargo 1.75.0 (1d8b05cdd 2023-11-20)
rustc 1.75.0 (82e1608df 2023-12-21)


neofetch:

██████████████████  ████████   leo@leo-hpenvyx360convertible13ay0xxx 
██████████████████  ████████   ------------------------------------- 
██████████████████  ████████   OS: Manjaro Linux x86_64 
██████████████████  ████████   Host: HP ENVY x360 Convertible 13-ay0xxx 
████████            ████████   Kernel: 6.5.13-7-MANJARO 
████████  ████████  ████████   Uptime: 35 mins 
████████  ████████  ████████   Packages: 1259 (pacman), 17 (flatpak) 
████████  ████████  ████████   Shell: bash 5.2.21 
████████  ████████  ████████   Resolution: 1920x1080 
████████  ████████  ████████   DE: Plasma 5.27.10 
████████  ████████  ████████   WM: kwin 
████████  ████████  ████████   Theme: [Plasma], Breeze [GTK2/3] 
████████  ████████  ████████   Icons: breeze [Plasma], breeze [GTK2/3] 
████████  ████████  ████████   Terminal: konsole
```

### Additional context

I've used and installed spacedrive before on the same machine and OS, a little while ago. Only now it seems to fail at reaching that one link "downloads.sentry-cdn.com/sentry-cli/2.21.2/sentry-cli-Linux-x86_64".

**Comment by brxken128** [comment]
The AUR package is not maintained by us, you're best off asking on the [page](https://aur.archlinux.org/packages/spacedrive). Alternatively, you could try installing `spacedrive-git` as that's another package in the AUR, but again this comes with the same note that we do not currently maintain packages in distro repositories.

**Comment by 0xBA5E64** [comment]
Adding my personal experience to this;
Spacedrive currently fails to build from source on Arch Linux due to certain Prisma binaries (`prisma-fmt`, `query-engine`, `migration-engine` & `introspection-engine`) that are currently only appear to be built for OpenSSL 3.0, whereas Arch Linux comes packaged with OpenSSL 3.2. I'd wager to guess that's also why the [`spacedrive-git`](https://aur.archlinux.org/packages/spacedrive-git) AUR package currently fails to build, as the error above appears to be identical to the one I experienced when compiling from source.

I personally managed to get around this by manually downloading the OpenSSL 3.0-builds of the respective Prisma binaries spewed out in the "Unable to download" errors and simply renaming them from "-3.0.x" to "-3.2.x". However this likely isn't an advisable way of taking care of this issue.

**Comment by owosoup** [comment]
Ah, thank you for posting this! I hope it'll be useful for others. Sorry for my lack of detail in the initial report, you filled in the gaps! I tried your method, it works great.

**Comment by HeavenVolkoff** [comment]
@0xBA5E64 `spacedrive-git` should be fixed after #1958 lands
